### ISSUE TITLE
Display content of "Description" on calendar view

### ISSUE THREAD

**Issue by trallnag** [kevinpapst/kimai2#2740]
**Is your feature request related to a problem? Please describe.**

Most views display the description field. The calendar view does not.

![image](https://user-images.githubusercontent.com/24834206/130974434-8db01738-9c8d-43a3-b9f2-ce887809cb5a.png)

**Describe the solution you'd like**

Show the content of the description field on respective entries on the calendar view. Cut it off if the description does not fit.

    ${activity_name} - ${description_trimmed}

Basically imitate Clockify:

![image](https://user-images.githubusercontent.com/24834206/130974154-43b42b0a-e4d4-4d21-a328-d435a2437531.png)

**Comment by pfrenssen** [comment]
#1719 is a feature request to allow descriptions using markdown in the calendar view, but this seems to me as something that would not add a lot of value and can cause problems. Markdown allows elements and text styles that might not fit in the limited space of the calendar items. Especially things like headings, images and tables might cause problems. But even minor text styling such as bold, underline or italic might be out of place here.

Wouldn't it be better to just display the description as plain text? Having visible markdown symbols in the description wouldn't be a problem for me personally, but it might also be nice to strip out the symbols and have just the plain text.

In my personal use the descriptions are very short but important, mainly used for the ticket number I am working on.

**Comment by kevinpapst** [comment]
The best would be, if the user could choose what is inside this tiny popup window. 
I discussed this topic with multiple people and everyone wants to see something else...

**Comment by noyez** [comment]
I post this with a little hesitation because you can easily break you perfectly working install of kimai2. Continue only if you're familiar with code and git and have plenty of backups. I believe that a generalized solution is much more work, but for a specific single need this is a simple change. I do something similar. I maintain a tiny patch-set for myself in git. It requires some skill and backups and it could completely mess-up your perfectly working installation, so approach w/ care. 

Enough of the disclaimer and on to the code, edit [user.html.twig](https://github.com/kevinpapst/kimai2/blob/a8ac6b2eda9de1c03c3a629d12f75257da64b004/templates/calendar/user.html.twig#L103).

```
                title: apiItem.activity.name + ' | ' + apiItem.description,
```

You'd then need re-run the composer and console part of the update process. 

Again, please only do this if you know what you're doing.
